smf/clickhouse/manifest.xml

```diff
@@ -16,13 +17,17 @@
       <envvar name="LD_LIBRARY_PATH" value="/opt/ooce/pgsql-13/lib/amd64" />
     </method_environment>
   </method_context>
+
   <exec_method type='method' name='start'
-    exec='ctrun -l child -o noorphan,regent /opt/oxide/clickhouse/clickhouse server --log-file /var/tmp/clickhouse-server.log --errorlog-file /var/tmp/clickhouse-server.errlog -- --path %{config/store} --listen_host %{config/listen_host} --http_port 8123 &amp;'
+    exec='/var/svc/manifest/site/clickhouse/method_script.sh'
```

Comment:
Method scripts ought not to go into the manifest tree, I would put this at `/opt/oxide/lib/svc/manifest/clickhouse.sh` instead.

Reply:
Moved